src/main/java/dk/kb/datahandler/kaltura/KalturaDeltaUploadJob.java

```diff
@@ -0,0 +1,304 @@
+    * <p>   
+    * 4) For each new stream:
+    * <p>
+    *  4.1) Upload the stream to kaltura. (Notice some streams do not have extension, but this seems not to be an issue with kaltura). The kaltura tag
+    *  for upload is 'delta-2025-05-01' where last part is current day.
+    * <p>
+    *  4.2) Update the record's kalturaid in ds-storage 
+    * <p>
+    * All records that has been updated with error or kalturaId will have mTime updated to new value.
+    * After completion the facade method will start a solr delta-index job. 
+    * 
+    * @param mTimeFrom Upload all streams for records in solr with mTimeFrom higher than this value. 
+    * @throws InternalServiceException If any Solr or Kaltura call fails. Stop uploading more. Maybe allow single kaltura upload jobs to fail later.
+    * 
+    */
+    
+    public static int uploadStreamsToKaltura(long mTimeFrom) throws InternalServiceException{
+
+        boolean moreSolrRecords = true;
+        long mTimeFromCurrent = mTimeFrom;
+        int numberStreamsUploaded=0;
+        String uploadTagForKaltura=getUploadTagForKaltura();  
+        //The minimumFileSizeInBytesvalue has been defined by Asger+Petur. It has been burned into the kaltura bulk upload and must not be changed, unless we start with a new empty kaltura partnerid.
+        //Next time I recommend a much higher value such as 4096 etc, this is a few seconds of audio.
+        long minimumFileSizeInBytes=700; 
+        String dsStorageUrl = ServiceConfig.getDsStorageUrl();
+        DsStorageClient storageClient = new DsStorageClient(dsStorageUrl);
+                        
+        while (moreSolrRecords) {
+            try {
+                SolrDocumentList docs = fetchSolrRecords(mTimeFromCurrent, 500);
+                if (docs.getNumFound() == 0) {
+                    moreSolrRecords = false; // End loop
+                }
+
+                for (SolrDocument doc : docs) {
+                   String resourceDescription=(String) doc.getFieldValue("resource_description");
+
+                   String title="";//Default
+                   ArrayList<String> titles=(ArrayList<String>) doc.getFieldValue("title"); //multivalue
+                   if (titles.size() >0) {
+                       title = titles.get(0);// take first
+                   }                   
+                   String description=(String) doc.getFieldValue("description");
+                   String fileId=(String) doc.getFieldValue("file_id");
+                   String id=(String) doc.getFieldValue("id");
+                   String originatesFrom= (String) doc.getFieldValue("originates_from");                                       
+                   long mTime= (Long) doc.getFieldValue("internal_storage_mTime");
+                   
+                   mTimeFromCurrent=mTime; //update mTime for next call                   
+                                 
+                 try {                                    
```

Comment:
The indentation of this try statement is a bit of, it should be on the same level as line 105 above

Reply:
Correct, also fixed indention in try block